pos_ask_vat/static/src/js/Misc/NumberBuffer.js

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright (C) 2022-Today KMEE (https://kmee.com.br)
+ License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+*/
+
+odoo.define("pos_ask_vat.NumberBuffer", function (require) {
+    "use strict";
+
+    var NumberBuffer = require("point_of_sale.NumberBuffer");
+    const {Gui} = require("point_of_sale.Gui");
+
+    const _updateBuffer = (input) => {
```

Comment:
as you are completely redefining this heavily used 70+ lines method, I think it would be better to explain why you did it and what you changed (possibly with explicit line comments). Otherwise it will be very hard to guess the side effects of this module and follow the Odoo updates...

Reply:
if the goal is to change only the VAT field buffer, isn't there a cleaner way or may be check field name? I'm no Javascript expert, but I still think this redefinition is a bit dirty. cc @legalsylvain any idea?